### PR TITLE
First draft of developer doc for Scim Group Push Beta

### PR DESCRIPTION
**Pull request by kasturikumthekar-okta**
Developer doc for Beta Scim Group Push. 
Resolves [OKTA-113016](https://oktainc.atlassian.net/browse/OKTA-113106)

Please review - @mystiberry-okta @parmstead-okta @bng-okta @tgu-okta

### CODE REVIEW

File: _source/_standards/SCIM/index.md [hunk 1]
```diff
@@ -738,6 +738,30 @@ Example:
...
+
+##### Read Group Details: GET /Groups/{id}
+
+With Group Push Beta, Okta now supports reading the Group's details by group id along with the membership details. If a Group is not found, your SCIM application may return a HTTP status 404("not found").For more details on the `/Groups/{id}` SCIM endpoint, see [section 3.4.1](https://tools.ietf.org/html/rfc7644#section-3.4.1) of the [SCIM 2.0 Protocol Specification](https://tools.ietf.org/html/rfc7644).
+
+
+##### Update Group Details: PUT /Groups/{id}
+
+With Group Push Beta, any updates to the Group profile and memberships in Okta can now be reflected into your SCIM application. Okta will do the following to make the Group changes effective:
+-   Make a GET request against `/Groups/{id}` on your SCIM API for the
```
Comment (bng-okta) [inline _source/_standards/SCIM/index.md]: For an update Group, do we do a GET first? I thought we only do the GET first for User Profile update but not Group update.  I don't see it in the ClientSession or maybe I missed it?

File: standards/SCIM/index.html [hunk 2]
```diff
@@ -1001,6 +1001,30 @@ <h5 id="get-groups-api-endpoint">GET /Groups API Endpoint</h5>
 </code></pre>
```
Comment (mystiberry-okta) [inline standards/SCIM/index.html]: Don't modify this file. It is the rendered version of index.md and is overwritten every time we publish.

File: _source/_standards/SCIM/index.md (hunk 1 above)
Comment (kasturikumthekar-okta) [inline _source/_standards/SCIM/index.md]: For updating the Group (`ScimClientSession.upsertGroup`), we do not GET the Group, however for updating the Group Memberships (for partial push - `ScimClientSession.updateGroupMembership`) , we do fetch the Group first via GET. @bng-okta

Comment (bng-okta) [inline _source/_standards/SCIM/index.md]: ah, okay.  thanks

**Comment by kasturikumthekar-okta** [comment]
@mystiberry-okta removed index.html from the commit.

**Pull request closed without merging**